pkg/operator/operator.go

```diff
@@ -94,6 +96,7 @@ func RunInstall(cmd *cobra.Command, args []string) {
 	util.KubeCreateSkipExisting(c.RoleBinding)
 	util.KubeCreateSkipExisting(c.ClusterRole)
 	util.KubeCreateSkipExisting(c.ClusterRoleBinding)
+	util.KubeCreateSkipExisting(c.SecurityContextConstraints)
```

Comment:
Actually is this where you wanted to add it at all? This only runs when calling `noobaa install` and is not running during reconcile at all. This is fine if we just want to support CLI install of the SCC, but this means that we still do not handle it instead of OCS operator.

Reply:
Also this will fail on non openshift clusters, where the SCC CRD is not available.
See how we create other openshift CR's like Routes.

Reply:
We want to install during cli install and that's it for now - reconciles will still be only in OCS operator
Tried in minikube and it worked fine. Just warned that the SCC is misssing: 
```INFO[0002] ❌ CRD Missing: SecurityContextConstraints "noobaa"```

Reply:
this looks like an error to me. that's why we used the ReconcileObjectOptional -
https://github.com/noobaa/noobaa-operator/blob/47283a1777a0a42567de44a608771de7981ca0aa/pkg/system/reconciler.go#L522
and the CheckOptional -
https://github.com/noobaa/noobaa-operator/blob/47283a1777a0a42567de44a608771de7981ca0aa/pkg/util/util.go#L366
to avoid these things looking like errors to the user.

Reply:
Added KubeCreateOptional